assets/src/edit-story/elements/image/display.js

```diff
@@ -29,20 +27,16 @@ const Img = styled.img`
 	${ ImageWithScale }
 `;
 
-function ImageDisplay( { id, src, origRatio, width, height, x, y, scale, focalX, focalY, rotationAngle, isFullbleed, forwardedRef } ) {
-	const elementProps = {
```

Comment:
Probably similar changes should be for other elements as well then, at least the Text? Not to have duplicate attributes on the element itself.

Reply:
Just noticed that you mentioned that not all the elements have been migrated yet.

Reply:
Yep. Wanted to first see if there were any objections to this approach. Will do migration before merging of course.